.jenkins/jenkins.sh

```diff
@@ -122,11 +122,9 @@ if grep -q "fv_dynamics" <<< "${script}"; then
 fi
 
 module load daint-gpu
-module add "${installdir}/modulefiles/"
-module load gcloud
-
+module load ${installdir}/modulefiles/gcloud/303.0.0
```

Comment:
is there a reason to switch this? I think the more general approach of adding the module directory as it allows us to load all the modules we custom made.

In the case where we only need the one it is also fine this way. Just curious

Reply:
I thought it didn't work anymore to module load the directory. I can check again

Reply:
Yes that still doesn't work -- please let me know the command if it works for you